en/docs/using-wso2-identity-server/account-Disabling.md

```diff
@@ -0,0 +1,156 @@
+# Account Disabling
+
+WSO2 Identity Server (WSO2 IS) enables an administrative user to disable
+a user account by configuring this feature in the WSO2 Identity Server
+and editing the user profile of the account. You can also disable
+(switch-off) the account disabling feature so that administrative users
+do not have permission to disable users.
+
+-   [Configuring WSO2 IS for account
+    disabling](#AccountDisabling-ConfiguringWSO2ISforaccountdisabling)
+-   [Disable an account](#AccountDisabling-Disableanaccount)
+-   [Sending email notifications for account
+    disabling](#AccountDisabling-Sendingemailnotificationsforaccountdisabling)
+
+### Configuring WSO2 IS for account disabling
+
+But first, you need to configure WSO2 Identity Server for user account
+locking and disabling. Follow the below steps to do this configuration.
+
+1.  Start the Identity Server and log into the management console using
+    your tenant credentials.
+2.  Click **Resident** under **Identity Providers** found in the
+    **Main** tab.
+3.  Expand the **Login Policies** tab.
+4.  Expand the **Account Disabling** tab and select the **Enable Account
+    Disabling** checkbox. Click **Update** to save changes.  
+    ![]( ../../assets/img/103330604/119113135.png) 
+
+    !!! tip
+    
+        Disable the account disabling feature
+    
+        To disable the account disabling option, unselect the **Enable
+        Account Disabling** in each of the tenants that you wish to disable
+        this option for.
+    
+        Alternatively, disable this feature for all tenants by doing the
+        following.
+    
+        1.  Open the `             identity-event.properties            `
+            file found in the
+            `             <IS_HOME>/repository/conf/identity            `
+            folder.
+    
+        2.  Set the following property to **false** to disable account
+            disabling. This will remove the account disabling option from
+            all tenants.
+    
+            ``` java
+                    account.disable.handler.enable=false
+        ```
+
+
+5.  Next, enable account locking. Expand the **Login Policies** tab.
+6.  Expand the **Account Locking** tab and select the **Account Lock
+    Enabled** checkbox. Click **Update** to save changes.  
+    ![]( ../../assets/img/103330604/119113139.png) 
+
```

Comment:
these image paths were already corrected but have been reverted back in this PR.

Reply:
Fixed.